qtdialogs/DlgSetupManager.py

```diff
@@ -0,0 +1,1268 @@
+               self,
+               self.tr('Invalid Core Settings'),
+               self.tr('Please select a valid scenario and network mode.')
+            )
+            return False
+         # Keep inputs disabled regardless
+         self.p2pPortInput.setEnabled(False)
+         self.rpcPortInput.setEnabled(False)
+         return True
+      except Exception:
+         return False
+
+   def _validateWalletsGroup(self):
+      try:
+         # Do not probe bridge readiness here; UI handles it during listing
+         return True
+      except Exception:
+         return True
+
+   def _validateDbGroup(self):
+      try:
+         db = self._collectDbGroup()
+         if db['scenario'] == 'Connect to Remote Database':
+            host = db['remoteHost']
+            portText = db['remotePort']
+            if not host or not portText:
+                QtWidgets.QMessageBox.warning(
+                   self,
+                   self.tr('Missing Remote DB Info'),
+                   self.tr('Please provide host and port for the remote database.')
+                )
+                return False
+            try:
+               port = int(portText)
+               if port < 1 or port > 65535:
+                  raise ValueError()
+            except ValueError:
+               QtWidgets.QMessageBox.warning(
+                  self,
+                  self.tr('Invalid Port'),
+                  self.tr('Remote DB port must be an integer between 1 and 65535.')
+               )
+               return False
+         return True
+      except Exception:
+         return False
+
+   # ----------------------
+   # Feature groups helpers
+   # ----------------------
+   def _collectPathsGroup(self):
+      """Return current paths from UI as a dict."""
+      return {
+         'core': self.satoshiHomePath.text(),
+         'armory': self.armoryDataDirEdit.text(),
+         'db': self.databaseDirEdit.text(),
+      }
+
+   def _collectCoreGroup(self):
+      """Return current core settings from UI as a dict."""
+      return {
+         'networkMode': self.networkModeCombo.currentText(),
+         'manageSatoshi': (self.scenarioCombo.currentIndex() == 0),
+      }
+
+   def _collectDbGroup(self):
+      """Return current database config from UI as a dict."""
+      dbScenario = self.databaseScenarioCombo.currentText()
+      dbTypeDisp = self.databaseTypeCombo.currentText() if dbScenario == 'Run Local Database' else ''
+      return {
+         'scenario': dbScenario,
+         'typeDisp': dbTypeDisp,
+         'remoteHost': self.remoteHostEdit.text() if dbScenario == 'Connect to Remote Database' else '',
+         'remotePort': self.remotePortEdit.text() if dbScenario == 'Connect to Remote Database' else '',
+         'remoteUser': self.remoteUserEdit.text() if dbScenario == 'Connect to Remote Database' else '',
+         'remotePass': self.remotePasswordEdit.text() if dbScenario == 'Connect to Remote Database' else '',
```

Comment:
there are no passwords to manipulate, the bridge will prompt you with an unlock callback if you try to load a locked peers db, same as with wallets